src/command-processor.js

```diff
@@ -36,8 +36,18 @@ export async function processMessage(msg) {
                 return "I have recorded your progress friend.";
             case STATS_CMD:
                 const pastWeights = await weightStats(msg.author.username);
-                const average = await averageWeight(pastWeights);
-                return `Past 7 Weights: ${pastWeights}\n7 Day Average: ${average}`;
+                const reversedWeights = pastWeights.reverse()
+                const currentAvg = averageWeight(reversedWeights.slice(-7));
+                const lastAvg = averageWeight(reversedWeights.slice(0,7));
+                const average = averageWeight(reversedWeights.slice(-7));
```

Comment:
Can we omit this? Seems to be the same value we calculate for `currentAvg`.

Reply:
Removed redundant variable.